Picture-elements markdown: Add `transform:none`. (#19147)
home-assistant/home-assistant.github.io: source/_lovelace/picture-elements.markdown

### How to use the style object

Position and style your elements using [CSS](https://developer.mozilla.org/en-US/docs/Web/CSS). More/other keys are also possible.

> Note, the default style for most elements includes [translate](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/translate)(-50%, -50%), which means that the coordinates you provide will set the position of the center of the element. If you wish to align to left use `transform: none`.

Reviewer: ```suggestion
Note, the default style for most elements includes [translate](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/translate)(-50%, -50%), which means that the coordinates you provide will set the position of the center of the element. Use `transform: none` to disable this behavior.
```

Maybe?
